components/payment_form/payment_form.scss

```diff
@@ -0,0 +1,93 @@
+.PaymentForm {
+    margin: 0 auto;
+    width: 75%;
+    max-width: 448px;
+
+    .form-row {
+        width: 100%;
+        margin-bottom: 24px;
+        display: flex;
+    }
+
+    .form-row-third-1 {
+        max-width: 288px;
+        width: 66%;
+        margin-right: 16px;
+    }
+    
+    .form-row-third-2 {
+        max-width: 144px;
+        width: 34%;
+    }
+
+    .section-title{
+        font-weight: 600;
+        font-size: 16px;
+        color: var(--center-channel-text);
+        text-align: left;
+        margin-bottom: 24px;
+    }
+
+    .selector{
+        font-size:14px;
+        
+    }
+
+    .full-width {
+        width: 100%
+    }
+
+    Input{
+        line-height: 40px;
+        font-size: 14px;
+    }
+
+    .PaymentForm-saved {
+        width: 442px;
+        height: fit-content; 
+        background: #FFFFFF;
+        border: 1px solid rgba(61, 60, 64, 0.08);
+        box-sizing: border-box;
+        border-radius: 4px;
+        padding: 24px;
+        margin-bottom: 16px;
+    }
+
+    .PaymentForm-saved-title {
+        font-family: Metropolis;
+        font-weight: 600;
+        font-size: 14px;
+        line-height: 18px;
+        letter-spacing: 1.5px;
+        text-transform: uppercase;
+        color: #22406D;
+        opacity: 0.4;
+        margin-bottom: 16px;
+    }
+
+    .PaymentForm-saved-card {
+        font-family: Source Sans Pro;
+        font-size: 16px;
+        line-height: 24px;
+        color: #22406D;
+        border-bottom: 1px solid rgba(0, 0, 0, 0.12);
```

Comment:
We need to change have this use the theme colors but can make it part of the saved card ticket: https://mattermost.atlassian.net/browse/MM-29096